extensions/amp-selector/0.1/amp-selector.js

```diff
@@ -67,6 +71,44 @@ export class AmpSelector extends AMP.BaseElement {
     }
   }
 
+  /** @override */
+  mutatedAttributesCallback(mutations) {
+    mutations.forEach(mutation => {
+      const newValue = mutation.newValue;
+
+      switch (mutation.name) {
+        case 'selected':
```

Comment:
could we not re-use the clickHandler_ (we could create a method `handleChenge_(element)`) for this case - would the mutation object have the target element triggering on which the change happens?